haystack/nodes/audio/answer_to_speech.py

```diff
@@ -0,0 +1,60 @@
+import logging
+from typing import Union, List, Dict, Any, Tuple
+
+import os
+import hashlib
+from pathlib import Path
+
+from espnet2.bin.tts_inference import Text2Speech
+import soundfile as sf
+
+from haystack.nodes import BaseComponent
+from haystack.schema import Answer, AudioAnswer, GeneratedAudioAnswer
```

Comment:
Exactly, I had that in mind. The node is definitely generating "speech", but in the future we can reuse these primitives for other types of audio answers, so we should keep their name generic. So I will keep it like this if you agree this is a reasonable idea

Reply:
I thought about Speech because right now the AudioAnswer node uses a Text2Speech model under the hood and therefore I found SpeechAnswer more suitable but if we indeed allow Question Answering via extracting a sound snippet from a longer AudioDocument in future then I agree that we can keep AudioAnswer. Similarly, ImageAnswer could be a subimage of a larger Image(Document) then. 
I am trying to imagine how we would generate an AudioAnswer that is not speech based on a TextAnswer. 
Could you maybe come up with an example where we might use an AudioAnswer in future that is not speech?

Reply:
Oh wow that sounds awesome! I would love to try that out one day. I have got a funny idea what to do with that. :D